apis/v1alpha1/generator.yaml

```diff
@@ -27,8 +27,6 @@ model_name: acm-pca
 prefix_config: {}
 resources:
   CertificateAuthority:
-    reconcile: 
-      requeue_on_success_seconds: 30
```

Comment:
why did this get removed?

My concern is that you create a CA and the state in ACK shows PENDING_CERTIFICATE for a veryyyy long time now despite the activation resource working successfully.